src/Cryptol/Eval/Generic.hs

```diff
@@ -1119,6 +1119,21 @@ errorV sym ty msg = case ty of
   TVAbstract {} -> cryUserError sym msg
 
 
+-- | Expect a word value.  Mask it an 8-bits ASCII value
```

Comment:
"Mask it"?

Reply:
"mask it to an 8-bit ASCII value" is the intended meaning